crates/devolutions-pedm-shared/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "devolutions-pedm-shared"
+version = "2024.8.2"
```

Comment:
suggestion: Internal crates that we don’t publish (with `publish = false`), are assigned the version `0.0.0` because it doesn’t really matter and we don’t want to keep updating that number at each release.